pkgs/top-level/all-packages.nix

```diff
@@ -10207,14 +10207,14 @@ in
   rust_1_45 = callPackage ../development/compilers/rust/1_45.nix {
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
   };
-  rust_1_48 = callPackage ../development/compilers/rust/1_48.nix {
```

Comment:
Do we need an alias for that?

Reply:
No. Rust 1.48 is removed by this and entirely replaced by 1.49. What would a rust_1_48  alias point to?

Reply:
It wouldn't point to anything but would throw that users do not get an eval error if they used rust 1.48 directly.

Reply:
We don't have this for rust 1.47 or 1.46 or 1.44 or any version before that. The error someone will get when trying to use rust_1_48 is good enough.
With a few exceptions there is no reason to refer to a specific Rust version, because the Rust team does a great job with their stability promises. They build _all_ crates in the crates.io registry and individually address anything that starts failing with a new Rust version with the authors.

Reply:
I think we have too many rust versions to add aliases for all of them. We mainly need specific versions for thunderbird/firefox.